go.mod

```diff
@@ -10,18 +10,19 @@ require (
 	github.com/beorn7/perks v1.0.1 // indirect
 	github.com/btcsuite/btcutil v0.0.0-20190425235716-9e5f4b9a998d
 	github.com/cespare/cp v1.1.1
+	github.com/coinbase/rosetta-sdk-go v0.3.4
 	github.com/davecgh/go-spew v1.1.1
 	github.com/davidlazar/go-crypto v0.0.0-20190912175916-7055855a373f // indirect
 	github.com/deckarep/golang-set v1.7.1
 	github.com/edsrzf/mmap-go v1.0.0 // indirect
-	github.com/ethereum/go-ethereum v1.8.27
+	github.com/ethereum/go-ethereum v1.9.18
```

Comment:
why update ethereum? any potential compatibility issue?

Reply:
Rosetta SDK depends on this version of eth. As it is not compatible with our version, I've added a replacement to our dependent version here: https://github.com/harmony-one/harmony/pull/3297/files#diff-37aff102a57d3d7b797f152915a6dc16R83